jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -97,45 +140,53 @@ def getLabelScore(label) {
 }
 
 //checks if current semver is lower than the supposed latest one
-def hasNewGARelease(currentVersion, newVersion) {
-    def current = currentVersion.replaceAll("[^0-9]", "").tokenize('.')
-    def latest = newVersion.replaceAll("[^0-9]", "").tokenize('.')
-
-    def currentMajor = current[0] as Integer
-    def latestMajor = latest[0] as Integer
-    def currentMinor = current[1] as Integer
-    def latestMinor = latest[1] as Integer
-    def currentPatch = current[2] as Integer
-    def latestPatch = latest[2] as Integer
-    def currentLabel = current[3]
-    def latestLabel = latest[3]
-
-    def previousVer = 0
-    def currentVer = 0
-    for (i = 0; i <= 2; i++) {
-        if (i > 0) {
-            previousVer = i - 1
-        }
-        currentVer = i
-
-        def previousDiff = current[previousVer] == latest[previousVer]
-        if (i == 0) {
-            previousDiff = true
-        }
-        def currentDiff = latest[currentVer] > current[currentVer]
-        if (previousDiff && currentDiff) {
-            return true
-        }
-    }
-
-    def currentLabelScore = getLabelScore(currentLabel)
-    def latestLabelScore = getLabelScore(latestLabel)
-
-    if (latestLabelScore > currentLabelScore) {
-        return true
-    }
-
-    return false
+def hasNewGARelease(currentVersion, newVersion, productName) {
+  def current = ""
+  def latest = ""
+
+  if (productName == "fuse") {
+    current = currentVersion.replaceAll("[^0-9/-]", "").tokenize("-")
```

Comment:
@mikenairn For fuse, since their version is tagged like this `application-templates-<major>-fuse-<minor>-<patch>` we can tokenize by `-` instead of `.` This should give us the correct `major`, `minor` and `patch` values.